classes/output/rating_system_configuration_renderer.php

```diff
@@ -52,17 +81,28 @@ public function render() {
         }
     }
 
+    /**
+     * Renders the rating configuration view
+     *
+     * @return bool|string
+     * @throws \moodle_exception
+     */
     private function render_configuration() {
         $html = $this->render_form();
-        return $this->renderer->render_from_template('capquiz/matchmaking_configuration', [
+        return $this->renderer->render_from_template('capquiz/rating_system_configuration', [
```

Comment:
Good catch 👍

The matchmaking_configuration and rating_system_configuration mustache templates are identical, hence we have not experienced any issues with this yet.